server/.gitignore

```diff
@@ -1,5 +1,7 @@
 # security
 .env
+services/firebaseServiceAccountKey.json
+config/creds.json
```

Comment:
keep the firebase crediental json file in one place, we should only need one of these lines